manifests/s/sandboxlab/HippoScan/1.6.6/sandboxlab.HippoScan.locale.ru-RU.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: sandboxlab.HippoScan
+PackageVersion: 1.6.6
+PackageLocale: ru-RU
+Publisher: sandboxlab
+PackageName: HippoScan

```

```suggestion
PackageName: HippoScan, âåðñèÿ
```
Must match Control Panel/Apps&Features

---

My bad, I did not realize it meant version. Thank you for the clarification.